### DISCUSSION TITLE
Only show today's values (from 0:00)

### DISCUSSION THREAD

**Discussion by nschoot** [kalkih/mini-graph-card#969] [Ideas]
I would like to be able to see todays gas consumption, electricity production, consumption etc, grouped by hour

I have defined a bar chart for this, it shows the consumption per hour.
It works all fine for me, except that the chart currently alway shows a fixed amount of bars, which is 24 hours in my case.

So, it always shows the last 24 hours, which is not what I want, because
a) I only care about today and 
b) it is hard to tell where in the chart "today" begins...

So, would it be possible to do either of this:
- (Preferred) Start the chart with values from today only (perhaps it could be an option of "hour24", which currently only supports true/false) and show 24 hours, starting at 0:00 and which contains northing for the hours of today that haven't happened yet.
- (Alternative) Make it possible to have an option for "hours_to_show" to show the number of hours of today (including the unfinished hour).

Current config
````
type: custom:mini-graph-card
show:
  graph: bar
hour24: true
group_by: hour
hours_to_show: 24
aggregate_func: diff
entities:
  - sensor.gasverbruik_vandaag
 ````

**Comment by ildar170975** [comment]
Same as https://github.com/kalkih/mini-graph-card/issues/726

Similar to:
https://github.com/kalkih/mini-graph-card/issues/77
https://github.com/kalkih/mini-graph-card/issues/363

Could be achieved by https://github.com/kalkih/mini-graph-card/issues/697

Currently could be achieved by https://github.com/kalkih/mini-graph-card/issues/697#issuecomment-1014930071
